Mage.Client/src/main/java/mage/client/dialog/CustomOptionsDialog.java

```diff
@@ -0,0 +1,289 @@
+package mage.client.dialog;
+
+import mage.constants.MultiplayerAttackOption;
+import mage.constants.RangeOfInfluence;
+import mage.game.match.MatchOptions;
+import mage.game.mulligan.MulliganType;
+import org.apache.log4j.Logger;
+
+import javax.swing.*;
+
+/**
+ * App GUI: custom options for match/tournament
+ *
+ * @author artemiswkearney
+ */
+public class CustomOptionsDialog extends MageDialog {
+
+    public enum SaveLoadKeys {
+        TABLE(
+                PreferencesDialog.KEY_NEW_TABLE_NUMBER_OF_FREE_MULLIGANS,
+                PreferencesDialog.KEY_NEW_TABLE_MULLIGAN_TYPE,
+                PreferencesDialog.KEY_NEW_TABLE_PLANECHASE
+        ),
+
+        TOURNEY(
+                PreferencesDialog.KEY_NEW_TOURNAMENT_NUMBER_OF_FREE_MULLIGANS,
+                PreferencesDialog.KEY_NEW_TOURNAMENT_MULLIGUN_TYPE,
```

Comment:
typo in variable name, also planechase could be renamed so table/tourney keys are consistent

Reply:
These were all like that when I found them (I believe there's both a "mulligun" and a "mullgian" in there somewhere, although that one may have been in one of the deleted UI element variable names). While I'm in favor of fixing typos, I think that belongs in a separate PR in case the refactor breaks something.